library/src/main/java/com/getstream/sdk/chat/adapter/MessageViewHolderFactory.kt

```diff
@@ -25,6 +25,8 @@ open class MessageViewHolderFactory {
         const val FILE_ATTACHMENT = 4
     }
 
+    lateinit var listenerContainer: ListenerContainer
```

Comment:
This is injected manually by `MessageListView`. It's public, as anyone extending `MessageViewHolderFactory` might need to access the listeners for their own custom ViewHolders.

Reply:
Could we have an "empty implementation" of this field instead of a `lateinit var`?

Reply:
It's guaranteed to be initialized by our own library code, we just can't do it at the time when the constructor is called. If it's somehow used before it's set, it _should_ crash, because that's an error in the init calls.